misc/services/procurement-webui/procurement-webui-backend/src/main/java/de/metas/procurement/webui/Application.java

```diff
@@ -81,12 +75,12 @@ public TaskExecutor asyncCallsTaskExecutor()
 		return executor;
 	}
 
-	@Bean
-	public Docket docket()
-	{
-		return new Docket(DocumentationType.OAS_30)
-				.select()
-				.paths(PathSelectors.any())
-				.build();
-	}
+	// @Bean
+	// public Docket docket()
```

Comment:
@mrie04 @metas-ts saw that swagger for procurement is commented out. is this a conscious decision or just a mistake?

Reply:
this one was a mistake, i'll adjust